src/CollectionsNg/Hamt.elm

```diff
@@ -429,8 +178,7 @@ set' shift hash key val ls =
                             hashPositionWithShift newShift xHash
 
                         subNodes =
-                            empty
-                                |> setByIndex collisionPos currValue
+                            setByIndex collisionPos currValue empty
```

Comment:
I (wholly subjectively) like the old style, and pipes have no runtime cost thanks to a compiler optimization.